specs/spec-report/aggregated_report.json

```diff
@@ -0,0 +1 @@
+{"com.electriccloud.plugin.spec.NMB27865":{"executedFeatures":["NMB-27865","plugin promotion"],"ignoredFeatures":[],"stats":{"failures":0,"errors":0,"skipped":0,"totalRuns":2,"successRate":1.0,"time":24500},"title":"","narrative":""},"com.electriccloud.plugin.spec.PluginTestHelper":{"executedFeatures":[],"ignoredFeatures":[],"stats":{"failures":0,"errors":0,"skipped":0,"totalRuns":0,"successRate":1.0,"time":1761},"title":"","narrative":""},"com.electriccloud.plugin.spec.NMB27865":{"executedFeatures":["NMB-27865","plugin promotion"],"ignoredFeatures":[],"stats":{"failures":1,"errors":0,"skipped":0,"totalRuns":2,"successRate":0.5,"time":8379},"title":"","narrative":""},"com.electriccloud.plugin.spec.PluginTestHelper":{"executedFeatures":[],"ignoredFeatures":[],"stats":{"failures":0,"errors":0,"skipped":0,"totalRuns":0,"successRate":1.0,"time":1981},"title":"","narrative":""}}
```

Comment:
files under specs/spec-report are test output. We don't need to check them in.

Reply:
Added to .gitignore
what about the specs/.gradle?

Reply:
yes, that could be ignored as well